connectors/cybereason/configurations/config.json

```diff
@@ -0,0 +1,26 @@
+{
+    "connection": {
+        "type": {
+            "type": "connectorType",
+            "displayName": "Cybereason"
+        }
+    },
+    "configuration": {
+        "auth": {
+            "type": "fields",
+            "cybereason_url": {

```

Comment:
We need to move this to connection, same as: https://github.com/opencybersecurityalliance/stix-shifter/blob/develop/stix_shifter_modules/cybereason/configuration/config.json

We need to follow same structure to that as much as possible.

Reply:
Updated config file as suggested